src/openapi/streaming/serializer/wrappers/protobuf-wrappers.js

```diff
@@ -7,8 +7,10 @@ export default {
                 },
 
                 toObject(message, options) {
-                    const seconds = Number(message.seconds) * 1000;
-                    const date = new Date(seconds);
+                    const { seconds, nanos } = message;
+
+                    // Date with support for nano precision
+                    const date = new Date((Number(seconds) * 1000) + (Number(nanos) / 1000000));

```

Comment:
Shall we adjust this to be an integer? e.g.

Math.floor(Number(nanos) / 1000000)

so its clear this is to get the ms, not that Date supports fractions..